examples/jsm/renderers/webgpu/WebGPUBindings.js

```diff
@@ -1,17 +1,20 @@
+import WebGPUStorageBuffer from './WebGPUStorageBuffer.js';
```

Comment:
It seems this import is not necessary.

Reply:
Never mind. I clean it up locally.

Reply:
87f564470765886606db25a5e94d8be01e2cdbef

Reply:
Thanks. I just woke up now.

Reply:
I sometimes forget that the project members are spread all over the world^^.